tests/integration_tests/clouds.py

```diff
@@ -274,7 +274,7 @@ def _mount_source(instance: LXDInstance):
             (cloudinit_path, "/usr/lib/python3/dist-packages/cloudinit"),
             (
                 os.path.join(cloudinit_path, "..", "config", "cloud.cfg.d"),
-                "/etc/cloud/cloud.cfg.d",
+                "/var/tmp/etc/cloud/cloud.cfg.d",
```

Comment:
If we're pushing these files directly, there's no reason to mount this directory anymore, correct?

Reply:
d'oh, leftover attempt to avoid the nobody/nogroup bin mount. I was originally trying to bind mount then copy the full directory from the bind-mount over to /etc/cloud/cloud.cfg.d, but I realized that I should just copy the files in the first place without the bind-mount anyway. So I'm dropping that mount defitinitions as it's unused now.